buddyboss/buddyboss-platform
# Group Message Fixies

### All Submissions:









### How to test the changes in this Pull Request:

1. User can delete their messages from group thread.
2. When user delete from system all of his thread needs to de deleted when in a thread there is no any other people sent the message.
3. When user delete from system all of his threads meesages need to be deleted and other peoples messages should remain as it is.

### Proof Screenshots or Video

https://www.loom.com/share/013a5c28c7f14b6f9c06e5d9032e03b9



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
